server/api/boards.go

```diff
@@ -127,40 +125,40 @@ func (a *API) handleCreateBoard(w http.ResponseWriter, r *http.Request) {
 
 	requestBody, err := ioutil.ReadAll(r.Body)
 	if err != nil {
-		a.errorResponse(w, r.URL.Path, http.StatusInternalServerError, "", err)
+		a.errorResponse(w, r, err)
 		return
 	}
 
 	var newBoard *model.Board
 	if err = json.Unmarshal(requestBody, &newBoard); err != nil {
-		a.errorResponse(w, r.URL.Path, http.StatusBadRequest, "", err)
+		a.errorResponse(w, r, model.NewErrBadRequest(""))
```

Comment:
I think the Unmarshal error would be helpful for troubleshooting.  

```suggestion
		a.errorResponse(w, r, model.NewErrBadRequest(err.Error()))
```